extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,9 +48,13 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type] > iframe {
```

Comment:
Hrm.  Yeah, this may be the only way to identify A4A FF slots ATM.  But I'm worried b/c we introduced that `data` attribute originally to aid in debugging.  I don't know that we're committing to keeping it around.  Is this the best selector you can use for this?